docs/tutorials/add-oauth2-provider.md

```diff
@@ -185,42 +206,18 @@ You can test your OAuth2 provider by trying to login using the [OAuth2 method](h
 
 Pass your new adapter name as the provider parameter. If login is successful, you will be redirected to your success URL parameter. Otherwise, you will be redirected to your failure URL.
 
-If everything goes well, raise a pull request and be ready to respond to any feedback which can arise during our code review.
+> Please note that you do not need to try to commit the changes you made in the `console` git submodule within appwrite, it was done only for testing purposes. Make a separate pull request in [`appwrite/console`](https://github.com/appwrite/console) repository with the required console changes as mentioned earlier.
```

Comment:
Once we include the step to create a new branch ( as I mentioned above ) we can remove this `note`